resources/js/common.js

```diff
@@ -283,3 +282,40 @@ function postwith (to,p) {
   document.body.removeChild(myForm) ;
 }
 
+
+function isValidDate(dateString)

```

Comment:
> By the way, would you mind testing and merging this PR before this PR becomes conflicting again?

Is there a test plan (instructions to setup and test) somewhere? So anyone could test it even without having followed this PR.